content/en/agent/kubernetes/apm.md

```diff
@@ -48,54 +60,39 @@ To enable APM trace collection in Kubernetes:
      # (...)
     ```
 
-{{% /tab %}}
-{{% tab "Helm" %}}
-
-**Note**: If you want to deploy the Datadog Agent as a deployment instead of a DaemonSet, configuration of APM via Helm is not supported.
-
-- Update your [datadog-values.yaml][2] file with the following APM configuration:
+- **If using an old agent version (7.17 or lower)**, set the `DD_APM_NON_LOCAL_TRAFFIC` and `DD_APM_ENABLED` variable to `true` in your *env* section of the `datadog.yaml` Agent manifest:
 
     ```yaml
-    datadog:
-      ## @param apm - object - required
-      ## Enable apm agent and provide custom configs
-      #
-      apm:
-        ## @param enabled - boolean - optional - default: false
-        ## Enable this to enable APM and tracing, on port 8126
-        #
-        #
-        enabled: true
-
-        ## @param port - integer - optional - default: 8126
-        ## Override the trace Agent DogStatsD port.
-        ## Note: Make sure your client is sending to the same UDP port.
-        #
-        port: 8126
+     # (...)
+          env:
+            # (...)
+            - name: DD_APM_ENABLED
+              value: 'true'
+            - name: DD_APM_NON_LOCAL_TRAFFIC
+              value: "true"
+     # (...)
     ```
 
-- Then upgrade your Datadog Helm chart.
-
 {{% /tab %}}
 {{< /tabs >}}
 
-    **Note**: On minikube, you may receive an `Unable to detect the kubelet URL automatically` error. In this case, set `DD_KUBELET_TLS_VERIFY=false`.
+**Note**: On minikube, you may receive an `Unable to detect the kubelet URL automatically` error. In this case, set `DD_KUBELET_TLS_VERIFY=false`.
 
 2. **Configure your application pods to pull the host IP in order to communicate with the Datadog Agent**: Use the downward API to pull the host IP; the application container needs the `DD_AGENT_HOST` environment variable that points to `status.hostIP`.
 
     ```yaml
-    apiVersion: apps/v1
-    kind: Deployment
-     # ...
-        spec:
-          containers:
-          - name: "<CONTAINER_NAME>"
-            image: "<CONTAINER_IMAGE>"/"<TAG>"
-            env:
-              - name: DD_AGENT_HOST
-                valueFrom:
-                  fieldRef:
-                    fieldPath: status.hostIP
+        apiVersion: apps/v1
+        kind: Deployment
+         # ...
+            spec:
+              containers:
+              - name: "<CONTAINER_NAME>"
+                image: "<CONTAINER_IMAGE>"/"<TAG>"
+                env:
+                  - name: DD_AGENT_HOST
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: status.hostIP
     ```
 
 3. **Configure your application tracers to emit traces**: Point your application-level tracers to where the Datadog Agent host is using the environment variable `DD_AGENT_HOST`. Refer to the [language-specific APM instrumentation docs][2] for more examples.
```

Comment:
Below for `DD_APM_ENABLED` and `DD_APM_NON_LOCAL`, might be worth mentioning "true by default for agent versions 7.18+"

Reply:
I think this is still not resolved, unless you disagree with doing this change.

Reply:
I could've sworn I added it. Weird o_O. Adding now...